src/jsx/components/input_info.jsx

```diff
@@ -77,7 +72,8 @@ class InputInfo extends React.Component {
               className="dropdown-menu"
               aria-labelledby="dropdown-menu-button"
             >
-              {on_datamonkey
+              {/* The options shown in the dropdown are highly platform dependent, hence the ternary sea below */}
+              {this.props.platform == "dataMonkey"
```

Comment:
Platform specific parameters is a bit sticky. It would be hard to know off hand what is different for each respective platform and we'd end up with a rash of conditionals throughout the codebase. I would rather provide options to display certain items. (e.g. options={{log="/log.txt"}).

Reply:
> Platform specific parameters is a bit sticky.

I can attest to this first-hand after working in this PR. I removed all platform specific parameters and just pass in props for fasta, analysisLog and originalFile.